packages/cli/commands/start.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
ref previous comments about file wide rule disablements

Reply:
For reference, `@typescript-eslint/no-non-null-assertion` is almost always because of mistyped DB collections. I know how to fix the typing but agreed with Omar to defer until we finish UM.

Reply:
More generally, I have been removing lint exceptions in the testing branches. Perhaps we should wait until those are merged before refactoring and/or adding project-wide exceptions.